packages/rule-meta-theme-color/README.md

```diff
@@ -0,0 +1,193 @@
+# Require a 'theme-color' meta tag with a valid value (`@sonarwhal/rule-meta-theme-color`)
+
+`rule-meta-theme-color` rule checks if a single `theme-color` meta tag
+is specified in the `<head>` with a valid value.
+
+## Why is this important?
+
+The [`theme-color` meta tag][spec] provides a way to suggest a color
+that browsers should use to customize the display of the page or of
+the surrounding user interface. For example, browsers might use the
+color for the page's title bar, or use it as a color highlight in a
+tab bar or task switcher.
+
+So, especially in the context of progressive web apps, for a more
+app-like feel, providing a theme color is essential.
+
+Note: Always specify the theme color using the meta tag. Even though
+it can also be declared in the [web app manifest file][manifest],
+browsers only acknowledge it from there once the user has added the
+page to their homescreen.
+
+## How to use this rule?
+
+To use it you will have to install it via `npm`:
+
+```bash
+npm install @sonarwhal/rule-meta-theme-color
+```
+
+Note: You can make `npm` install it as a `devDependency` using the
+`--save-dev` parameter, or to install it globally, you can use the
+`-g` parameter. For other options see [`npm`'s
+documentation](https://docs.npmjs.com/cli/install).
+
+And then activate it via the [`.sonarwhalrc`][sonarwhalrc]
+configuration file:
+
+```json
+{
+    "connector": {...},
+    "formatters": [...],
+    "parsers": [...],
+    "rules": {
+        "meta-theme-color": "error"
+    },
+    ...
+}
+```
+
+## What does the rule check?
+
+The rule checks if a single `theme-color` meta tag is specified in the
+`<head>` and the value of its `content` attribute is a [valid CSS
+color][color] supported by the [targeted
+browsers](https://sonarwhal.com/docs/user-guide/further-configuration/browser-context/).
+
+### Examples that **trigger** the rule
+
+The `theme-color` meta tag is not specified:
+
+```html
+<!doctype html>
+<html lang="en">
+    <head>
+        <title>example</title>
+        ...
+    </head>
+    <body>...</body>
+</html>
+```
+
+<!-- markdownlint-disable MD038 -->
+
+The `theme-color` meta tag is wrongly specified as ` theme-color`:
```

Comment:
Maybe add a note that there's an space at the beginning to make it clear?

Reply:
Yes, please, the fist time I though it didn't support the short version of the color hehehe.